exosphere-shared/src/docker-helper.ls

```diff
@@ -84,5 +85,11 @@ class DockerHelper
       child_process.exec-sync 'docker rm -f $(docker ps -aq)'
 
 
+  # Prints the last n number of lines from a given text
+  @_print-last-lines = (text, number-of-lines) ->
+    text
+      .split \\n
```

Comment:
Such hackery leads to all sorts of unpredictable bugs, messes, and headaches. Even if it works in this particular example, the moment you change something it might no longer work, meaning because we don't do things properly here, other changes will have unintended side effects. 

Here are a few of the fun things waiting for you in the future:
- lines now have a dangling `\r` while being processed. If you print a line for debugging, it yanks your cursor to the beginning of the line in the terminal, and all output is getting messed up, overwriting text. Fun times.
- when joining things back together, the last line might end up only with a `\r`, without an `\n`. 
- if you decide to sanitize code, for example via `trim`, then it suddenly doesn't work on Windows anymore

Because of all this crazy shit, it's better to do things properly right from the start. Simple, straightforward, robust, easy to understand code that isn't full of hidden trap doors and issues that are just waiting to blow up in your face, possibly in other parts of the code base.